libc-test/build.rs

```diff
@@ -3137,6 +3137,7 @@ fn test_linux(target: &str) {
                "netdb.h",
                "netinet/in.h",
                "netinet/ip.h",
+               //"netinet/sctp.h",
```

Comment:
This is because the header is too new here, right? If so, a FIXME comment would helpful when upgrading CI images.

Reply:
just not necessary at the moment.